src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -231,16 +231,24 @@ function useChatUtilsHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatUtilsHook {
     });
   };
 
-  const helperGetChannelDetail = async (channel: ChannelList) => {
+  const helperGetChannelDetail = async (channel: ChannelList, lastChatDate?: string) => {
     if (!localDb) return;
 
     try {
       let response;
       if (channel?.channelType === 'ANON_PM') {
-        response = await AnonymousMessageRepo.getAnonymousChannelDetail('messaging', channel?.id);
+        response = await AnonymousMessageRepo.getAnonymousChannelDetail(
+          'messaging',
+          channel?.id,
+          lastChatDate
+        );
       } else {
         const channelType = channel?.channelType === 'GROUP' ? 'group' : 'messaging';
-        response = await SignedMessageRepo.getSignedChannelDetail(channelType, channel?.id);
+        response = await SignedMessageRepo.getSignedChannelDetail(
+          channelType,
+          channel?.id,
+          lastChatDate
+        );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are generally good, adding an optional `lastChatDate` parameter to the `helperGetChannelDetail` function and passing it to the `getAnonymousChannelDetail` and `getSignedChannelDetail` functions. However, there is a potential issue with error handling. If an error occurs during the execution of `getAnonymousChannelDetail` or `getSignedChannelDetail`, the error will be caught but not handled or logged. This could make debugging difficult if an error occurs. Consider logging the error or rethrowing it after logging.

```diff
     } catch (e) {
+      console.error(e);
     }
```
This change will ensure that any errors that occur during the execution of these functions are logged to the console, making them easier to debug.


<!-- This is an auto-generated comment by OSS CodeRabbit -->